packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Providers/TestData/ModelCustomizationTests/TestCustomization_CanChangeModelName.cs

```diff
@@ -0,0 +1,24 @@
+#nullable disable
+
+using System;
+
+namespace NewNamespace
+{
+    [AttributeUsage(AttributeTargets.Class)]
+    internal class CodeGenTypeAttribute : Attribute
```

Comment:
Can we leave a todo here referencing https://github.com/Azure/autorest.csharp/issues/4551 since if they are public APIs this goes away?

Reply:
Sure